components/infobox/commons/infobox_league.lua

```diff
@@ -251,8 +251,9 @@ function League:_setLpdbData(args, links)
 		endpatch = args.endpatch or args.epatch,
 		type = args.type,
 		organizers = mw.ext.LiquipediaDB.lpdb_create_json(
-			League:_resolveAllRedirectsInTable(
-				League:_getNamedTableofAllArgsForBase(args, 'organizer')
+			Table.mapValues(
+				League:_getNamedTableofAllArgsForBase(args, 'organizer'),
```

Comment:
The previous function had a nil check, so surely:

```suggestion
				League:_getNamedTableofAllArgsForBase(args, 'organizer') or {},
```

Reply:
Hm, it looks like _getNamedTableofAllArgsForBase always return an empty table at the very least